### PR TITLE
Add android support, adjust shake params

### PR DESCRIPTION
**Pull request by juxd**
building previews to test android support.

@hzxcaryn how should we handle non-support cases?

### CODE REVIEW

**Comment by indocomsoft** [comment]
<3 seems to work, just need to calibrate the aylol increase

**Comment by juxd** [comment]
Can handle this in the menu/lobby? Then if no permissions given then kick LOL.

**Comment by juxd** [comment]
ok, ready for merge! will work on graphics on top of this.

File: Web/src/games/BalloonShake.tsx [hunk 1]
```diff
@@ -3,67 +3,93 @@ import {
...
 import useCounter from './hooks';
+import { unwrap } from './utils/rxhelpers';
+import Button from '../components/common/Button';
 
-async function getShakeObservable(): Promise<Observable<number>> {
-  if (!window.DeviceMotionEvent) {
+enum MotionPermission {
+  NOT_SET = 0,
+  DENIED = 1,
+  GRANTED = 1
```
Comment (afterdusk) [inline Web/src/games/BalloonShake.tsx]: Is there a reason the ints used to represent are the same? :O

Comment (juxd) [reply Web/src/games/BalloonShake.tsx]: o shit
lol typo

**Comment by juxd** [comment]
@afterdusk @indocomsoft are there other changes for this? because i got another PR waiting on top of this

**Pull request merged**